iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/CatalogConstants.java

```diff
@@ -15,6 +15,8 @@ public final class CatalogConstants {
   /** Used to uniquely identify an update towards a table from user side. */
   public static final String COMMIT_KEY = "commitKey";
 
+  static final String STOP_TABLE_CREATION_TOGGLE = "stop_create";
```

Comment:
can we make all toggle keys to follow same pattern name? maybe FEATURE_TOGGLE_STOP_CREATE, FEATURE_TOGGLE_S3_STORAGE etc

Reply:
addressed it.